ONSdigital/ssdc-rm-docker-dev
# Add rh docker dev

# Motivation and Context

We want to bring the RH services into docker dev.
# What has changed



rh-services.yml and rh-dev.yml created to hold the RH docker services and dependant containers.
Makefile has been updated so that there are targes for `up`, `down`, `pull` and `logs` for RH and RM individually, as well as for both at the same time.
# How to test?



Test the make targets and ensure everything comes up and down as required.
Note, if bring up both RM and RH services, and then stop one or the other, the pubsub emulator will not stay up so you will need to run the `*-up` target for the services you wanted to stay up.

Run the ATs (will not work with the RH stuff up)
Run the RH cucumber tests ([with changes on the linked branch of the same name](https://github.com/ONSdigital/sdc-int-rh-cucumber/pull/28))
# Links



https://trello.com/c/zPH8WtWD/3401-bring-rh-into-our-docker-compose-setup8
# Screenshots (if appropriate):
